test/Test/Pos/Block/Logic/Mode.hs

```diff
@@ -136,7 +137,7 @@ instance Buildable TestParams where
             _tpStakeDistribution
             _tpStartTime
       where
-        utxo = _tpGenUtxo & \(GenesisUtxo u) -> u
+        utxo = _tpGenTxpContext & \g -> unGenesisUtxo (g ^. gtcUtxo)
```

Comment:
Without pattern matching such construct doesn't make any sense. You can write `unGenesisUtxo (_tpGenTxpContext ^. gtcUtxo)`.